### ISSUE TITLE
Upcoming Command not working

### ISSUE THREAD

**Issue by inishchith** [Cloud-CV/evalai-cli#104]
```sh
$ evalai challenges upcoming
```
Doesn't work . 
Instead the function that should be invoked is named `future`. [REF.](https://github.com/Cloud-CV/evalai-cli/blob/5a2f67e60ed1e3b146a1081059c0ee66768fce76/evalai/challenges.py#L99) 
```sh
$ evalai challenges future
```

@RishabhJain2018 @isht3 this is just a `typo` kind of issue. let me know if i can propose a quick fix. 😄

**Comment by isht3** [comment]
@inishchith Yup, definitely a typo. Please feel free to start working on a fix! Thanks a lot!

**Comment by inishchith** [comment]
@isht3 replacing `future` ( function name ) with upcoming with `upcoming` would work just fine, i feel. 
There are many files with `future` reference, so changing every occurrence isn't a good idea because even the  API `urls` has a `/future` reference .
What do you think?

**Comment by isht3** [comment]
@inishchith I feel that the keyword future is more compatible and consistent across all of EvalAI and EvalAI CLI. But I definitely agree that `upcoming` sounds more verbose when it comes to the command. It's definitely up for discussion. @RishabhJain2018 What do you think?

**Comment by inishchith** [comment]
@RishabhJain2018 Any suggestions? 😄
